### PR TITLE
Fixes #2073. Add combined member signature tests

### LINKED ISSUE #2073: Add tests for Combined Member Signatures
11.1 Combined Member Signatures is not covered by co19 tests. Need to add a coverage

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: Language/Interfaces/Combined_Member_Signature/combined_member_signature_A03_t08.dart [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  void set m4(Object o) {}
+  void set m5(String s) {}
+}
+
+abstract class C implements I1, I2 {}
+
+mixin M implements I1, I2 {}
+
+test(C c, M m) {
+  c.#m1.expectStaticType<Exactly<void Function(num)>>();
```
Comment (chloestefantsova) [inline Language/Interfaces/Combined_Member_Signature/combined_member_signature_A03_t08.dart:56]: Should `#` be removed?

**Review by chloestefantsova — CHANGES_REQUESTED** [review]
LGTM with one suggestion.

File: Language/Interfaces/Combined_Member_Signature/combined_member_signature_A03_t08.dart (hunk 1 above)
Comment (sgrekhov) [reply Language/Interfaces/Combined_Member_Signature/combined_member_signature_A03_t08.dart]: Ups. Thank you for noticing! Setter cannot be torn-off, so the entire test is wrong. Deleted

Comment (chloestefantsova) [reply Language/Interfaces/Combined_Member_Signature/combined_member_signature_A03_t08.dart]: Thanks!

**Review by chloestefantsova — APPROVED** [review]

**Pull request merged**